server/api/characterApi.js

```diff
@@ -1,14 +1,15 @@
-import get from "axios";
+import {axiosService} from "../services/axiosService.js";
```

Comment:
Add eslint!
```suggestion
import { axiosService } from "../services/axiosService.js";
```
U can add also the base path and turn the path to `services/axiosService` + Omit suffix `.js`